awx/main/models/__init__.py

```diff
@@ -12,7 +12,16 @@
 from awx.main.models.organization import Organization, Profile, Team, UserSessionMembership  # noqa
 from awx.main.models.credential import Credential, CredentialType, CredentialInputSource, ManagedCredentialType, build_safe_env  # noqa
 from awx.main.models.projects import Project, ProjectUpdate  # noqa
-from awx.main.models.inventory import CustomInventoryScript, Group, Host, Inventory, InventorySource, InventoryUpdate, SmartInventoryMembership  # noqa
+from awx.main.models.inventory import (  # noqa
+    CustomInventoryScript,
+    Group,
+    Host,
+    HostMetric,
+    Inventory,
+    InventorySource,
+    InventoryUpdate,
+    SmartInventoryMembership,
+)  # noqa
```

Comment:
Remove this `# noqa`, the `# noqa` on line 15 does what is wanted.

Reply:
I have made the changes.